src/sentry/tasks/integrations/__init__.py

```diff
@@ -54,7 +54,6 @@ def should_comment_sync(
 
 from .create_comment import create_comment
 from .kick_off_status_syncs_impl import kick_off_status_syncs
-from .migrate_issues import migrate_issues
```

Comment:
does this need to be removed?

Reply:
it causes circular dependency, plan is to delete all these imports so other files so specific ones.